dashboard/app/controllers/api/v1/projects/public_gallery_controller.rb

```diff
@@ -5,11 +5,21 @@ def index
       expires_in 5.seconds, public: true # cache
     end
 
-    render json: ProjectsList.fetch_published_projects(
-      params[:project_type],
-      limit: params[:limit],
-      published_before: params[:published_before]
-    )
+    if params[:showFeatured] == "1"
```

Comment:
rather than use a special value, you could just test `if params[:showFeatured]`, which would be less error prone.